controllers/vsl.go

```diff
@@ -122,41 +122,38 @@ func (r *DPAReconciler) ValidateVolumeSnapshotLocations(dpa oadpv1alpha1.DataPro
 			Spec: *vslSpec.Velero,
 		}
 
+		// check if VSL name is specified in DPA spec
+		if vslSpec.Name != "" {
+			vsl.Name = vslSpec.Name
+		}
+
 		// check for valid provider
 		if vslSpec.Velero.Provider != AWSProvider && vslSpec.Velero.Provider != GCPProvider &&
 			vslSpec.Velero.Provider != Azure {
 			r.Log.Info("Non-supported provider specified, might be a misconfiguration")
```

Comment:
I think we should remove these logs, OR, change them to be level error, since we are erroring it out now

Reply:
ack

Reply:
only this log was left inside `ValidateVolumeSnapshotLocations` function, do not forget to remove it :)

Reply:
oops. done.